Source/WebCore/platform/mediastream/cocoa/IncomingAudioMediaStreamTrackRendererUnit.cpp

```diff
@@ -90,10 +90,10 @@ void IncomingAudioMediaStreamTrackRendererUnit::addSource(Ref<AudioSampleDataSou
             return;
 
         m_outputStreamDescription = outputDescription;
-        m_audioBufferList = makeUnique<WebAudioBufferList>(m_outputStreamDescription);
-        m_sampleCount = m_outputStreamDescription.sampleRate() / 100;
+        m_audioBufferList = makeUnique<WebAudioBufferList>(*m_outputStreamDescription);
```

Comment:
It seems we are almost always using a WebAudioBufferList next to a outputDescription.
Maybe we should have WebAudioBufferList own a CAAudioStreamDescription and use a getter to its CAAudioStreamDescription when we need it. This does not need to be done in this patch.